packages/zendesk-client/package.json

```
@@ -0,0 +1,32 @@
+{
+	"name": "@automattic/zendesk-client",
+	"description": "Manages the connection to the Zendesk API",
+	"author": "Automattic Inc.",
+	"main": "dist/cjs/index.js",
+	"module": "dist/esm/index.js",
+	"types": "dist/types",
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"version": "1.0.0",
+	"sideEffects": false,
+	"scripts": {
+		"build": "tsc --build ./tsconfig.json ./tsconfig-cjs.json",
+		"clean": "tsc --build ./tsconfig.json ./tsconfig-cjs.json --clean && rm -rf dist",
```

Building before cleaning is not needed IMO. It cleans the other packages unnecessarily.

```suggestion
		"clean": "rm -rf dist",
```